bin/ci-stack.ts

```diff
@@ -17,7 +17,11 @@ const defaultEnv: string = 'Dev';
 
 const ciConfigStack = new CIConfigStack(app, `OpenSearch-CI-Config-${defaultEnv}`, {});
 
-const ciStack = new CIStack(app, `OpenSearch-CI-${defaultEnv}`, {});
+const ciStack = new CIStack(app, `OpenSearch-CI-${defaultEnv}`, {
+  env: {
+    region: 'us-east-1',
```

Comment:
This is required while enabling VPC Flow logs. Without this build will succeed but cdk targets will fail.